src/core/scene/a-scene.js

```diff
@@ -308,10 +308,10 @@ module.exports = registerElement('a-scene', {
     render: {
       value: function (time) {
         var camera = this.camera;
-        var timeDelta = time - this.time;
```

Comment:
actually time and timeDelta kind of mix when reading and typing. it's not optimizing for typing efficiency but to avoid confusing both by making look different.